test/e2e/tests/confirmations/header.spec.js

```diff
@@ -0,0 +1,88 @@
+const { strict: assert } = require('assert');
+const {
+  defaultGanacheOptions,
+  openDapp,
+  unlockWallet,
+  WINDOW_TITLES,
+  withFixtures,
+} = require('../../helpers');
+const FixtureBuilder = require('../../fixture-builder');
+
+describe('Header component', function () {
+  if (!process.env.ENABLE_CONFIRMATION_REDESIGN) {
+    return;
+  }
+
+  const SIGNATURE_CONFIRMATIONS = [
+    { name: 'Personal Sign signature', testDAppBtnId: 'personalSign' },
+    { name: 'Sign Typed Data signature', testDAppBtnId: 'signTypedData' },
+    { name: 'Sign Typed Data v3 signature', testDAppBtnId: 'signTypedDataV3' },
+    { name: 'Sign Typed Data v4 signature', testDAppBtnId: 'signTypedDataV4' },
+    { name: 'Sign Permit signature', testDAppBtnId: 'signPermit' },
+  ];
+
+  SIGNATURE_CONFIRMATIONS.forEach((confirmation) => {
+    it(`${confirmation.name} component includes header with balance and copyable address element`, async function () {
```

Comment:
I appreciate both are simple tests, but for the sake of modularity and separate failures, should these be separate tests?